File: gsb/gsb-web/src/main/webapp/gsb/gardian/js/device.form.part.js
System.Declare("com.gongsibao.gardian.web");
com.gongsibao.gardian.web.DeviceFormPart = org.netsharp.panda.commerce.FormPart.Extends({

    ctor: function () {
        this.base();
    },
    deviceTypeChange:function (newValue, oldValue) {

        if(newValue==1||newValue==0){
            $("fieldset:first").next().show();
        }else{
            $("fieldset:first").next().hide();
        }
    }

});

File: gsb/gsb-gardian/src/main/java/com/gongsibao/gardian/web/DeviceFormPart.java
package com.gongsibao.gardian.web;

import com.gongsibao.gardian.base.IDeviceService;
import org.netsharp.communication.ServiceFactory;
import org.netsharp.panda.commerce.FormPart;

public class DeviceFormPart extends FormPart {
    IDeviceService deviceService= ServiceFactory.create(IDeviceService.class);
}
